conf/openmetadata.yaml

```diff
@@ -296,6 +296,7 @@ eventMonitoringConfiguration:
   batchSize: ${EVENT_MONITOR_BATCH_SIZE:-10}
   pathPattern: ${EVENT_MONITOR_PATH_PATTERN:-["/api/v1/tables/*", "/api/v1/health-check"]}
   latency: ${EVENT_MONITOR_LATENCY:-[0.99, 0.90]} # For value p99=0.99, p90=0.90, p50=0.50 etc.
+  servicesHealthCheckInterval: ${EVENT_MONITOR_SERVICES_HEALTH_CHECK_INTERVAL:-300}
```

Comment:
why are we adding this config under pipelineService, this has nothing to do with pipeline right?
Lets introduce healthCheck configuration